score_test.go

```diff
@@ -842,6 +842,103 @@ func TestScoreRetention(t *testing.T) {
 	}
 }
 
+func TestScoreResetTopicParams(t *testing.T) {
```

Comment:
This test only seems to be checking recapping; maybe rename the test appropriately? With the current name, it seems like it's doing a different thing.

Reply:
ok

Reply:
done